### PR TITLE
Wishlist management

### PR DESCRIPTION
**Pull request by rdthetha**
Files to check:
Wishlist.jsx
Wishlistcontext.js
WIshlistCard.jsx

### CODE REVIEW

**Comment by kaustubh-jsr** [comment]
The code looks good. Could not find any major issues in it. Although you might want to take a look at the navigation bar which does not take up 100% view port width at large screens. Rest looks good to me.

File: src/context/wishlistContext.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React ,{createContext,useContext, useReducer} from 'react'
+import { DataTest } from '../data';
```
Comment (gauravsinhaweb) [inline src/context/wishlistContext.js]: data should come from backend

File: src/pages/Wishlist/Wishlist.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import WishlistCard from '../../components/Card/WishlistCard'
+import NavigationTop from '../../components/Navigation/navigation-top'
```
Comment (gauravsinhaweb) [inline src/pages/Wishlist/Wishlist.jsx:3]: folder name should be in lowercase

**Review by gauravsinhaweb — APPROVED** [review]
good to merge after changes

File: src/context/wishlistContext.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+const WishlistContext=createContext();
+
+const WishlistProvider=({children})=>{
+
+    const WishlistReducer=(state,action)=>{
```
Comment (eakansh125) [inline src/context/wishlistContext.js]: For better refactoring, reducer function can be moved to a separate file

**Review by eakansh125 — APPROVED** [review]
Added comments for refactoring.
UX suggestion:
You can remove the pricing card from Cart page if the cart is empty.
Looks good to be merged

File: src/context/wishlistContext.js (hunk 1 above)
Comment (rdthetha) [reply src/context/wishlistContext.js]: Done! Thank you!

**Pull request merged**